src/SectionMeetings.tsx

```diff
@@ -0,0 +1,163 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React from 'react';
+import * as Moment from 'moment';
+import { IMeeting } from './types';
+
+interface Props {
+  meetings: Array<IMeeting>;
+}
+
+const styles = {
+  container: {
+    flex: 1,
+    wordWarp: 'normal',
+    justifyContent: 'flex-start',
+    div: {
+      display: 'inline-block',
+    },
+  },
+  boldedText: {
+    fontWeight: 700,
+  },
+};
+
+const SectionMeetings: React.FC<Props> = ({ meetings }) => {
+  // Correct for null meeting arrays
+  let correctedMeetings = meetings ?? [];
+  let sortedSectionMeetings = correctedMeetings.sort((a, b) =>
+    Moment.default(a.beginDate).isBefore(Moment.default(b.beginDate)) ? 1 : -1
+  );
+  console.log(sortedSectionMeetings[1]);
+  return (
+    <div css={styles.container}>
+      {sortedSectionMeetings.length === 1 && (
+        <div>
+          <div css={styles.boldedText}>
+            {Moment.default(sortedSectionMeetings[0].beginDate).format('dddd')}s
+          </div>
+          {' from '}
+          <div css={styles.boldedText}>
+            {Moment.default(sortedSectionMeetings[0].beginDate).format(
+              'h:mm A'
+            )}
+          </div>
+          {' to '}
+          <div css={styles.boldedText}>
+            {Moment.default(sortedSectionMeetings[0].beginDate)
+              .add(sortedSectionMeetings[0].minutesDuration, 'minutes')
+              .format('h:mm A')}
+          </div>
+        </div>
+      )}
+      {/* Sections with two identical meetings a week */}
+      {sortedSectionMeetings.length === 2 &&
```

Comment:
Could we pull this out? We have logic that basically renders a different component for one meeting, two meetings and more than two meetings. How about we do something like:

```
if (sortedSectionMeetings.length === 1) {
  return <div css={styles.container}>
....
}

if (sortedSectionMeetings.length === 2) {
  if (Moment.default(sortedSectionMeetings[0].beginDate).format('h:mm') ===
          Moment.default(sortedSectionMeetings[1].beginDate).format('h:mm') &&
        sortedSectionMeetings[0].minutesDuration ===
          sortedSectionMeetings[1].minutesDuration) {
    return ....
  }
  return ....
}
return ....
```

That way we're a little more explicit about our different states and the logic isn't embedded in the JSX